test/gtest/common/test_ucc.h

```diff
@@ -237,6 +248,16 @@ class UccReq {
     static void startall(std::vector<UccReq> &reqs);
 };
 
+#define DATA_FINI_ALL(_test, _ctx) for (auto &c : ctxs) { _test->data_fini(c); }
+
+#define CHECK_REQ_NOT_SUPPORTED_SKIP(_UccReq, _action) do{  \
+        if ((_UccReq).status == UCC_ERR_NOT_SUPPORTED) {    \
+            _action;                                        \
+            GTEST_SKIP();                                   \
+        }                                                   \
+        ASSERT_EQ(UCC_OK, (_UccReq).status);                \
+    } while(0)
+
 void clear_buffer(void *_buf, size_t size, ucc_memory_type_t mt, uint8_t value);
 
 #define PREDEFINED_DTYPES \
```

Comment:
missing UCC_DT_BFLOAT16

Reply:
PREDEFINED_TYPES macro is only used with collectives that do not do any reductions. Do you think we need to add?

Reply:
yes, why not, we anyway test different datatypes and I don't think we test bfloat16 copy operation somewhere else

Reply:
added